src/core/app.lua

```diff
@@ -246,6 +275,16 @@ function main (options)
       breathe = latency:wrap_thunk(breathe, now)
    end
 
+   if options.ingress_drop_monitor then
```

Comment:
Let's default to on :)  i.e. `if options.ingress_drop_monitor or options.ingress_drop_monitor == nil then`, as above with the latency measurer.